common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1138,6 +1140,17 @@
                 }
             },
 
+            listenForDragDrop: function() {
+                var captions = document.querySelector('.closed-captions'),
+                    draggable;
+
+                if (typeof Draggabilly === "function" && captions) {
+                    draggable = new Draggabilly(captions, { containment: true });
+                } else {
```

Comment:
It's unclear why you can assume that the reason the if statement is False is that draggabilly isn't there.  If the captions element is always present, then it shouldn't need to be part of that check.  If it isn't, then this error message could be reporting an incorrect reason for the missing functionality.